projects/packages/post-list/composer.json

```
@@ -0,0 +1,53 @@
+{
+	"name": "automattic/jetpack-post-list",
+	"description": "PostList",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"require": {},
+	"require-dev": {
+		"automattic/jetpack-changelogger": "^1.2"
+	},
+	"autoload": {
+		"files": [
+			"actions.php"
+		],
+		"classmap": [
+			"src/"
+		]
+	},
+	"repositories": [
+		{
+			"type": "path",
+			"url": "../*",
+			"options": {
+				"monorepo": true
+			}
+		}
+	],
+	"scripts": {
+		"build-development": [
+			"Composer\\Config::disableProcessTimeout",
+			"pnpm run build"
+		],
+		"build-production": [
+			"Composer\\Config::disableProcessTimeout",
+			"pnpm run start"
```

Shouldn't it be the opposite?

```suggestion
			"Composer\\Config::disableProcessTimeout",
			"pnpm run start"
		],
		"build-production": [
			"Composer\\Config::disableProcessTimeout",
			"pnpm run build"
```

---

Using "pnpm run start" in "build-development" causes `jetpack build packages/post-list` to run endlessly. I'm not sure why.